ui/admin/tests/acceptance/aliases/update-test.js

```diff
@@ -101,12 +90,12 @@ module('Acceptance | aliases | update', function (hooks) {
 
     urls.alias = `${urls.aliases}/${instances.alias.id}`;
 
-    await click(`[href="${urls.aliases}"]`);
-    await click(DROPDOWN_BUTTON_SELECTOR);
+    await click(commonSelectors.HREF(urls.aliases));
+    await click(selectors.DROPDOWN_BUTTON_SELECTOR);
 
-    assert.dom(DROPDOWN_ITEM_SELECTOR).exists();
-    assert.dom(CLEAR_DROPDOWN_SELECTOR).exists();
-    await click(CLEAR_DROPDOWN_SELECTOR);
+    assert.dom(selectors.DROPDOWN_ITEM_SELECTOR).exists();
```

Comment:
This assertion is not necessary. As per name test, we are checking the user have the option to clear an alias this selector is returning 2 elements (possible false possitive), and the first selector that returns is the "Edit" option on the dropdown, not the "Clear".